examples/the-ast-typing-problem-with-polymorphic-records.flix

```diff
@@ -3,12 +3,12 @@
 /// We begin with a grammar for expressions.
 /// The definition of Exp is polymorphic in a record type that
 /// allows the AST to be decorated with additional fields.
-enum Expr[r: Record] {
+enum Expr[r: RecordRow] {
```

Comment:
I don't want to put `Record`, since we already use that term for values of _type_ `Record` of _kind_ `Type`.
I don't want to put `Row`, since that conflicts with rows for schemas, which are not the same type.

PureScript has (the equivalent to) `Row[Type]` because they have polymorphic kinds. I don't think we want to go there.
Other languages don't have other uses for the term "row" so they can just say `Row`.

But I agree that it's not ideal. I'll take suggestions :)

Reply:
OK, we will stick with `RecordRow` and `SchemaRow` for now.